launch_testing/launch_testing/launch_test.py

```diff
@@ -110,7 +111,11 @@ def main():
 
     # This is a list of TestRun objects.  Each run corresponds to one launch.  There may be
     # multiple runs if the launch is parametrized
-    test_runs = LoadTestsFromPythonModule(test_module, name=args.test_name + '.launch_tests')
+    test_runs = LoadTestsFromPythonModule(
+        test_module, name='{}.{}.launch_tests'.format(
+            args.package_name, test_file_basename
```

Comment:
Will this give silly names if `--package-name` is not specified?

Reply:
Shouldn't, see [line 102](https://github.com/ros2/launch/pull/228/files/b5cb01b7b1c34ac87092e79ccfc820bf3f7d47b5..f45718033494a0af5436db760bfb41dd3dd3980d#diff-2d26f84766e090142a8f854e7ee25e2bR102).